### ISSUE TITLE
How to Download VGG-19 model, we use it to initialize the first 10 layers for training

### ISSUE THREAD

**Issue by salvador-blanco** [ZheC/Realtime_Multi-Person_Pose_Estimation#174]
Where should I put the `VGG_ILSVRC_19_layers_deploy.prototxt` fiel, or how to configure it ?
I am puting in the `../training` folder is ti ok ?

**Comment by yw155** [comment]
Hi @salvador-blanco, actually you can put it everywhere, you just need to change the file path in the file of 'train_pose.sh'. Thanks.

**Comment by salvador-blanco** [comment]
@yw155  Thank you for the response, indeed in  the "train_pose.sh" there is a path to the `VGG_ILSVRC_19_layers.caffemodel` file:

`/home/gtk/m_HDD/caffe_train/build/tools/caffe train --solver=pose_solver.prototxt --gpu=$1 --weights=../../../model/vgg/VGG_ILSVRC_19_layers.caffemodel 2>&1 | tee ./output.txt`

 But in the file's name I got from the [git-hub](https://gist.github.com/ksimonyan/3785162f95cd2d5fee77) is a bit different `VGG_ILSVRC_19_layers_deploy.prototxt`

Again thank you for your help, I really appreciate it

**Comment by soans1994** [comment]
@yw155 @salvador-blanco 
Can you help me with finetuning the model with custom dataset. I have prepared the custom dataset and annotation file, and i can train the model freshly using vgg layers as you mentioned.  But when i tried the same appraoch for finetuning with preatrained coco model, the loss is high and the inference/detection is very bad. I used the following command.

caffe_train/build/tools/caffe train --solver=pose_solver.prototxt --gpu=$1 --weights=../../../model/coco/pose_iter_440000.caffemodel 2>&1 | tee ./output.txt

should i do any other changes before training the model? i follwed the same procedure as the author to generate the training files.

Thank you
